src/plugins/fieldflow/fieldflow.js

```diff
@@ -685,6 +685,11 @@ var componentName = "wb-fieldflow",
 				}
 			}
 		}
+		if ( isInline ) {
+			radCheckOut = "<div class='gc-form'>" + radCheckOut;
```

Comment:
I though the initial intend was to manually add the CSS class in a parent container of fieldflow

Reply:
The intention is any pages using fieldflow will be automatically applied so that those pages will not require updating.